### PR TITLE
[8.x] Update the Pluralizer Inflector instanciator

### PR DESCRIPTION
**Pull request by cesarep**
Uses the InflectorFactory::createForLanguage function to generate the instance

Should make it easier to change to other available languages rulesets such as French, Norwegian Bokmal, Portuguese, Spanish and Turkish

### CODE REVIEW

File: src/Illuminate/Support/Pluralizer.php [hunk 1]
```diff
@@ -136,14 +135,7 @@ public static function inflector()
...
         if (is_null($inflector)) {
-            $inflector = new Inflector(
-                new CachedWordInflector(new RulesetInflector(
-                    English\Rules::getSingularRuleset()
-                )),
-                new CachedWordInflector(new RulesetInflector(
-                    English\Rules::getPluralRuleset()
-                ))
-            );
+            $inflector = InflectorFactory::createForLanguage('english')->build();
```
Comment (seriquynh) [inline src/Illuminate/Support/Pluralizer.php:138]: I think we should use their language constants.

```php
<?php

use Doctrine\Inflector\InflectorFactory;
use Doctrine\Inflector\Language;

$inflector = InflectorFactory::createForLanguage(Language::ENLISH)->build();

```

Comment (cesarep) [reply src/Illuminate/Support/Pluralizer.php:138]: My main goal actually was to make it easier to change the language ruleset, just like changing the locale in the `config/app.php` file, adding another configuration for the pluralizer ruleset. I was thinking something like:
```php
$inflector = InflectorFactory::createForLanguage(  $app['config']->get('app.pluralizer_ruleset', 'english')  )->build();
```
But I'm not sure how to fully implement it...
Other classes appear to get the config values from `$this->app`, however class Pluralizer does not extends any other class, so `$this` shoudn't have access to the app right?

**Comment by cesarep** [comment]
I included the `use Illuminate\Support\Facades\Config;` on top, and
`$inflector = InflectorFactory::createForLanguage(Config::get('app.pluralizer_ruleset', 'english'))->build();`
 
and added `'pluralizer_ruleset' => 'portuguese',` to the `config/app.php` to test on my machine, it appers to be working fine!

I just had to mannualy set the users table name on the `User` Model

Should i make another pull request or is there a way to reopen it? I didn't expect it to be closed so quickly 😅

For me it's a very useful feature, not having to manually set a lot of parameters and table names because of different language plurals

**Comment by driesvints** [comment]
@cesarep a facade can't be mixed here. This class may be used outside Laravel. 

If you aren't ready with a PR then please use draft PR's to mark them as in progress.

**Pull request merged**